xtensa-lx/build.rs

```diff
@@ -1,6 +1 @@
-use std::{env, path::PathBuf};
-
-fn main() {
-    let out_dir = PathBuf::from(env::var("OUT_DIR").unwrap());
-    println!("cargo:rustc-link-search={}", out_dir.display());
-}
+fn main() {}
```

Comment:
we can just remove the `build.rs` now

Reply:
ah forgot about the links key - maybe a comment why we have an empty build script wouldn't hurt (not really needed of course)